index.js

```diff
@@ -252,9 +252,11 @@ class LedgerBridgeKeyring extends EventEmitter {
     // Note also that `getMessageToSign` will return valid RLP for all transaction types, whereas the
```

Comment:
This comment seems inaccurate :thinking:

I thought the whole point of the following condition is that `getMessageToSign` _doesn't_ return valid RLP for legacy transactions.

Reply:
@Gudahtt reading through this and the [conversation here](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/96#discussion_r697008656) I think you are right. We might want some clarification from @danjm but I don't know how quickly that will come since he's out til next Tuesday. Do you think we can move forward with this PR otherwise?

Reply:
Sure, we can fix the comment later